core/lib/refinery/core/environment_checker.rb

```diff
@@ -0,0 +1,54 @@
+# Check that Rails application configuration is consistent with what we expect.
+require 'pathname'
+
+module Refinery
+  module Core
+    class EnvironmentChecker
+
+      def initialize(root_path, environments: %w(development test production))
+        @root_path = Pathname.new(root_path)
+        @environments = environments
+      end
+
+      def call
+        environment_files.each do |env_file|
+          # Refinery does not necessarily expect action_mailer to be available as
+          # we may not always require it (currently only the authentication extension).
+          # Rails, however, will optimistically place config entries for action_mailer.
+          current_mailer_config = mailer_config(env_file)
```

Comment:
This mailer stuff should be done by refinerycms-inquiries. The core doesn't send mail. 

Maybe this whole thing shouldn't even be our concern?